src/content/docs/browser/new-relic-browser/configuration/proxy-agent-requests.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Proxy agent chunk requests and analytics data
+tags:
+  - Browser monitoring
+  - Proxy
+metaDescription: "The agent can be configured to fetch its code and send analytics through a proxy server instead of New Relic's endpoints."
+---
+
+From [version 1.240.0](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) and higher, the browser agent officially allows a proxy url to be set when determining either or both of:
+
+1. Where to fetch or lazy-load agent code
+2. Where to send observation data
+
+Today, many ad-blockers and security VPNs block client requests to third-party domains including New Relic's. As the number of internet users opting into the use of ad-blockers increase, customers will continue to lose observability on those devices. One solution would be to proxy those requests though an unblocked middleman. For example, `yoursite.com` can send its user observability data through its `nrbeacon.yoursite.com` proxy which would not be blocked by ad-blockers being that it's considered a first-party (sub)domain.
```

Comment:
```suggestion
As ad blockers and security VPNs increasingly block client requests to third-party domains, including New Relic's, you will see your observability on these devices decline. One way to get around ad blockers is to proxy your requests through a third-party domain that is not blocked. For example, `yoursite.com` could send user observability data through a proxy domain called `nrbeacon.yoursite.com`. This would work because ad blockers typically do not block first-party subdomains.
```

Reply:
"proxy your requests through a **third-party** domain that is not blocked." - the bolded part is incorrect

Reply:
Ah, it's suppose to be first-party domain, right? @cwli24